Packs/Iboss/Integrations/Iboss/Iboss_test.py

```diff
@@ -0,0 +1,278 @@
+"""Base Integration for Cortex XSOAR - Unit Tests file
+
+Pytest Unit Tests: all funcion names must start with "test_"
+
+More details: https://xsoar.pan.dev/docs/integrations/unit-testing
+
+MAKE SURE YOU REVIEW/REPLACE ALL THE COMMENTS MARKED AS "TODO"
+
+You must add at least a Unit Test function for every XSOAR command
+you are implementing with your integration
+"""
+
+# import demistomock as demisto
+# import CommonServerPython
+# from CommonServerPython import urljoin
+# import pytest
+# import json
+# import requests_mock
```

Comment:
You can delete those imports if they aren't used.

Reply:
Unused imports deleted